### PR TITLE
Fixed gke-to-gke-vpn demo and added more validation

### PR DESCRIPTION
**Pull request by emailbob**
Added more validation and made the cluster creation 3 times faster.

Fixed an error in vpn creation for the gke-to-gke-vpn demo

The install and cleanup scripts are now idempotent

### CODE REVIEW

**Comment by emailbob** [comment]
test this please

**Comment by emailbob** [comment]
test this please

**Comment by chrislovecnm** [comment]
Hey your git is not configured correctly. You are going to have to update the SHAs.

File: Jenkinsfile [hunk 1]
```diff
@@ -114,11 +116,12 @@ spec:
       }
 
     stage('gke-to-gke-vpn-validate') {
+          sleep 120
```
Comment (chrislovecnm) [inline Jenkinsfile]: Instead of sleeping we need to poll the cluster.

File: gke-to-gke-vpn/install.sh [hunk 2]
```diff
@@ -21,102 +21,119 @@
...
+  { echo >&2 "I require kubectl but it's not installed. Aborting."; exit 1; }
 
 ### Obtain current active PROJECT_ID
 PROJECT_ID=$(gcloud config get-value project)
 if [ -z "$PROJECT_ID" ]
-	then echo >&2 "I require default project is set but it's not.  Aborting."; exit 1;
+  then echo >&2 "I require default project is set but it's not. Aborting."; exit 1;
+fi
+
+### Ensure that the Forwarding rules quota is met
```
Comment (chrislovecnm) [inline gke-to-gke-vpn/install.sh:39]: This is so cool!!  Can we have this in all of our projects :)

**Comment by emailbob** [comment]
test this please

**Review by chrislovecnm — APPROVED** [review]

**Pull request merged**